tests/acceptance/fixtures.py

```diff
@@ -579,23 +579,30 @@ def test_foo():
 
 
 @pytest.fixture(autouse=True)
-def only_with_distro_feature(request, bitbake_variables):
-    """Fixture that enables use of `only_with_distro_feature(feature1, feature2)` mark.
+def only_with_mender_feature(request, bitbake_variables):
+    """Fixture that enables use of `only_with_mender_feature(feature1, feature2)` mark.
     Example::
 
-       @pytest.mark.only_with_distro_feature('mender-uboot')
+       @pytest.mark.only_with_mender_feature('mender-uboot')
        def test_foo():
            # executes only if mender-uboot feature is enabled
            pass
 
     """
 
-    mark = request.node.get_closest_marker("only_with_distro_feature")
+    mark = request.node.get_closest_marker("only_with_mender_feature")
     if mark is not None:
         features = mark.args
-        current = bitbake_variables.get("DISTRO_FEATURES", "").strip().split()
+        current = bitbake_variables.get("MENDER_FEATURES", "").strip().split()
+        # For historical reasons we also check in DISTRO_FEATURES.
+        current += bitbake_variables.get("DISTRO_FEATURES", "").strip().split()
         if not all([feature in current for feature in features]):
             pytest.skip(
                 "no supported distro feature in {} "
                 "(supports {})".format(", ".join(current), ", ".join(features))
             )
+
+
+# Deprecated.
+def only_with_distro_feature(request, bitbake_variables):
```

Comment:
But this is not actually a fixture, how is it supposed to work for the other branches?